.github/workflows/ci.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
     if: (!startsWith(github.head_ref, 'renovate/') && !startsWith(github.head_ref, 'dependabot/'))
     services:
       buildkitd:
-        image: moby/buildkit:v0.12.5
+        image: moby/buildkit:v0.13.0
```

Comment:
should we hold on this for some time? until v0.13.1 at least?

Reply:
seemed to work :sweat_smile:

Reply:
I have no doubt about this repository, but I'm more concerned with other repos, sensitive to reproducibility

Reply:
reverted